src/mame/konami/konamim2.cpp

```diff
@@ -1288,51 +1287,54 @@ ROM_START( btltryst )
 	ROM_REGION16_BE( 0x80, "eeprom", 0 )
 	ROM_LOAD( "93c46.7k",  0x000000, 0x000080, CRC(cc2c5640) SHA1(694cf2b3700f52ed80252b013052c90020e58ce6) )
 
-	ROM_REGION( 0x2000, "m48t58", 0 ) /* timekeeper SRAM */
+	ROM_REGION( 0x2000, "m48t58", 0 ) // timekeeper SRAM
 	ROM_LOAD( "m48t58", 0x000000, 0x002000, CRC(71ee073b) SHA1(cc8002d7ee8d1695aebbbb2a3a1e97a7e16948c1) )
 
 	DISK_REGION( "cdrom" )
 	DISK_IMAGE_READONLY( "636jac02", 0, SHA1(d36556a3a4b91058100924a9e9f1a58983399c6e) )
 ROM_END
 
-#if 0
+/*
```

Comment:
Don’t do this.  Disabled code will still be pre-processed, and comments are reformatted by srcclean to not use tabs beyond the indent level of the opening delimiter.

Reply:
Done, thanks! Just to learn... Why would we need a commented-out ROM definition to be preprocessed?

Reply:
Because if it’s at least preprocessed, it at least has to contain valid C++ tokens, so it catches certain kinds of code rot earlier.  Also editors and tools like MAME’s srcclean treat disabled code differently to comments.